sts/client.go

```diff
@@ -31,8 +33,10 @@ import (
 )
 
 const (
-	Namespace = "oasis:names:tc:SAML:2.0:assertion"
-	Path      = "/sts/STSService"
+	Namespace     = "oasis:names:tc:SAML:2.0:assertion"
+	BasePath      = "/sts"
+	Path          = BasePath + "/STSService"
+	SystemSTSPath = BasePath + "/system-STSService/sdk"
```

Comment:
Already in `sts` package, so just `SystemPath` for the name is good. And if not used outside the package, `systemPath`.